pkgs/top-level/python-packages.nix

```diff
@@ -11603,6 +11603,7 @@ in {
   jrnl = buildPythonPackage rec {
```

Comment:
can you also move this package from  `python-packages.nix` to a dedicated file and add a reference to `all-packages.nix`. It is not a library and therefor does not belong here.